nuxeo-coldstorage-web/elements/nuxeo-coldstorage-infobar.js

```diff
@@ -87,7 +87,12 @@ class ColdStorageInfobar extends mixinBehaviors([FiltersBehavior, FormatBehavior
   }
 
   _contentStoredInColdStorage(doc) {
-    return this.hasFacet(doc, 'ColdStorage') && doc.properties && doc.properties['coldstorage:coldContent'];
```

Comment:
the ftest are ok now,
but we have a problem
 - move the file to coldstorage 👍 
 - restore from coldstorage 👍 
 - move the file to coldstorage 👎 
for example
![Screenshot 2021-07-01 at 14 39 53](https://user-images.githubusercontent.com/59877775/124125698-37aecd00-da7a-11eb-8e78-67505b678834.png)

Reply:
![Screenshot 2021-07-01 at 14 42 27](https://user-images.githubusercontent.com/59877775/124126096-97a57380-da7a-11eb-8eaf-fe58ca4a3925.png)

Reply:
The send to coldstorage button is not related to the infobar. With the screenshot you provided, if toBeRestored is true, then the sendtocoldstorage button will not show up because you're still restoring the file.